static/app-strings.json

```diff
@@ -2040,5 +2040,11 @@
   "Tip Creators": "Tip Creators",
   "Only select creators can receive tips at this time": "Only select creators can receive tips at this time",
   "The payment will be made from your saved card": "The payment will be made from your saved card",
+  "Trending for #Art": "Trending for #Art",
+  "Trending for #Education": "Trending for #Education",
+  "Trending for #Technology": "Trending for #Technology",
+  "Watch content and earn more Credits for each level unlocked! 10 views required for level 1 (Current Score: 0). Only up to 10 views per day count.": "Watch content and earn more Credits for each level unlocked! 10 views required for level 1 (Current Score: 0). Only up to 10 views per day count.",
+  "Follow your favorite creators and earn more Credits for each level unlocked! Follow 1 creators for level 1 (Current Score: 0).": "Follow your favorite creators and earn more Credits for each level unlocked! Follow 1 creators for level 1 (Current Score: 0).",
+  "Gain a following to earn this reward. 1 validated followers needed for level 1 (Current Score: 0). This amount may not match your total followers.": "Gain a following to earn this reward. 1 validated followers needed for level 1 (Current Score: 0). This amount may not match your total followers.",
```

Comment:
These score-based entries should be removed as well as they can't be translated (values preprocessed before hitting the `__()` macro.  I'll try to fix by removing the macro for these later.